packages/rn-tester/metro.config.js

```diff
@@ -23,6 +23,7 @@ const config = {
   // Make Metro able to resolve required external dependencies
   watchFolders: [
     path.resolve(__dirname, '../../node_modules'),
+    path.resolve(__dirname, '../../scripts/build'), // babel-register.js
```

Comment:
Oh wait, `@react-native/asset-registry` is not a Node package — it's consumed by runtime code in `packages/react-native/Libraries/`!

**This probably means we don't need the `babel-register` setup at all** (just the root `.js.flow` files redirecting to `src/`) — since all modules loaded by React Native go through `@react-native/babel-preset`.

This is technically new ground. I think we want to define a new build `target` of `'react-native'` in `scripts/build/config.js`, configuring this to build files using `@react-native/babel-preset` as its Babel config. Will reach out via DM.